modules/odc-discovering-and-identifying-operator-backed-bindable-services.adoc

```diff
@@ -0,0 +1,13 @@
+:_content-type: PROCEDURE
+[id="odc-discovering-and-identifying-operator-backed-bindable-services_{context}"]
+== Discovering and identifying Operator-backed bindable services
+
+As an operator provider or a user who creates backing services, you must know which services are bindable before you create a backing service. The *Developer* perspective helps you discover and identify such bindable services. 
+
+.Procedure
+To discover and identify Operator-backed bindable services, consider the following alternative approaches:
+
+* Click *+Add* -> *Developer Catalog* -> *Operator Backed* to see the Operator-backed tiles. Operator-backed services that support service binding features have a *Bindable* badge on the tiles.
+* On the left pane of the *Operator Backed* page, select the *Bindable* checkbox.
+** Optional: Click on the help icon next to *Service binding* to see more information about bindable services.
+* Click  *+Add* -> *Add* and search for Operator-backed services. When you click on the service that is bindable, you can view the *Bindable* badge in the side panel to the right.
```

Comment:
```suggestion

* To discover and identify Operator-backed bindable services, consider the following alternative approaches:
+
* Click *+Add* -> *Developer Catalog* -> *Operator Backed* to see the Operator-backed tiles. Operator-backed services that support service binding features have a *Bindable* badge on the tiles.
* On the left pane of the *Operator Backed* page, select the *Bindable* checkbox.
** Optional: Click the help icon next to *Service binding* to see more information about bindable services.
* Click  *+Add* -> *Add* and search for Operator-backed services. When you click the service that is bindable, you can view the *Bindable* badge in the side panel to the right.
```

Reply:
Or consider converting these to conditional steps, as shown below. https://github.com/openshift/openshift-docs/pull/47760#discussion_r921122694

Reply:
Hello, @rolfedh 
Thanks for all the suggestions. They are very helpful. I will be incorporating all of them except this feedback. I have tried a different approach. Also, regarding line 12 in your comment, I am retaining my version to keep the technical accuracy intact.